src/Image.cc

```diff
@@ -100,8 +100,7 @@ NAN_METHOD(Image::New) {
  */
 
 NAN_GETTER(Image::GetComplete) {
-  Image *img = Nan::ObjectWrap::Unwrap<Image>(info.This());
-  info.GetReturnValue().Set(Nan::New<Boolean>(Image::COMPLETE == img->state));
+  info.GetReturnValue().Set(Nan::New<Boolean>(true));
```

Comment:
This is more correct than it was, but from the spec and testing Firefox, it looks like we want to return `false` when `src` is set but the image is still loading.

Appreciate the tests, clear commit message and references ❤️

Reply:
> it looks like we want to return `false` when `src` is set but the image is still loading.

Because loading is synchronous in node-canvas, an Image is never in a loading state.